LICENSE

```diff
@@ -1,2 +1,2 @@
 YEAR: 2024
-COPYRIGHT HOLDER: ggplot2 core developer team
+COPYRIGHT HOLDER: ggplot2 authors
```

Comment:
ggplot2 is special and should keep the prior wording

Reply:
I didn't know, but then we should omit `use_mit_licence()` in future upkeep issues.